dotnet/SK-dotnet.sln

```diff
@@ -992,6 +1003,12 @@ Global
 		{95163AA2-1ED5-412A-990B-C40B81934BFD}.Publish|Any CPU.Build.0 = Debug|Any CPU
 		{95163AA2-1ED5-412A-990B-C40B81934BFD}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{95163AA2-1ED5-412A-990B-C40B81934BFD}.Release|Any CPU.Build.0 = Release|Any CPU
+		{6F591D05-5F7F-4211-9042-42D8BCE60415}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{6F591D05-5F7F-4211-9042-42D8BCE60415}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{6F591D05-5F7F-4211-9042-42D8BCE60415}.Publish|Any CPU.ActiveCfg = Debug|Any CPU
+		{6F591D05-5F7F-4211-9042-42D8BCE60415}.Publish|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
```suggestion
		{6F591D05-5F7F-4211-9042-42D8BCE60415}.Publish|Any CPU.ActiveCfg = Publish|Any CPU
		{6F591D05-5F7F-4211-9042-42D8BCE60415}.Publish|Any CPU.Build.0 = Publish|Any CPU
```

Reply:
Thanks! This change is not required for unit test projects since we are not publishing them. Taking into account that it's `Debug` for other unit test projects, I will leave it as it is